lib/sovren-rest/client.rb

```diff
@@ -70,12 +70,26 @@ def headers
 
     ##
     # Builds up body of message for remote call.
-    def parse_body(input_file)
-      {
+    def parse_body(input_file, filemeta)
+      body = {
         'DocumentAsBase64String' => Base64.encode64(input_file),
         'OutputHtml' => 'true',
         'Configuration' => @configuration
       }
+
+      if filemeta.key?(:revision_date)
```

Comment:
This is probably overkill since we only have 1 extra "parameter" right now, but one pattern I've seen with extra "options" objects in other gems is to get the values by deleting them from the originally provided object:

```ruby
body['RevisionDate'] = filemeta.delete(:revision_date)
```

and then raising an exception after you are finished if the object isn't empty, so you can notify the caller what keys were provided that aren't supported.